go/v1/api/validators/vulnerability/vulnerability.go

```diff
@@ -116,19 +116,32 @@ func ValidateDetails(d *vulnpb.Details) []error {
 func validatePackageIssue(p *vulnpb.PackageIssue) []error {
```

Comment:
FixAvailable also needs to be validated.

Reply:
fixAvailable should always have the default value (false) since it is a bool, so I dont think we need validation there.